index.php

```diff
@@ -169,8 +169,9 @@
 			</div>
 			<div class="header-breadcrumbs">
 				<ul>
-					<li><a href="<?php echo $quickdownload_link?>" title="get the latest release"><?php echo $template['index']['quickdnl']?></a></li>
+					<li><a href="https://kvirc.d00p.de/appveyor/KVIrc-4.9.1-dev-git-6120-g9368f12-x86_64.exe" title="Get the Latest Release"><?php echo $template['index']['quickdnl']?></a></li>
```

Comment:
Eh... I don't exactly think we should consider this the "latest release", let alone force people to download the 64 bit version of it on windows.